web/client/examples/viewer/actions/fullScreen.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const ACTIVATE_BUTTON = 'ACTIVATE_BUTTON';
```

Comment:
The action has a global scope, so use a better name for this, like "TOGGLE_FULLSCREEN"